packages/backend/src/links/links.service.ts

```diff
@@ -27,12 +27,29 @@ export class LinksService {
   }
 
   async findAll(params: SearchLink) {
+    if (typeof params.term === 'undefined') return this.prisma.link.findMany()
```

Comment:
Seems good to me, although I think we need to add await (e.g. line 30) to each prisma function call, hence we used async functions.

Reply:
Fair, im not sure, how did it work without them...

Reply:
Because we return the response right away, it's not actually needed, Nest will wait for the promise to resolve before sending the HTTP response. But it's probably a good practice to always write await before prisma calls, so you won't forget them when they're required (when you want to do something with the result)